### DISCUSSION TITLE
urgent help please about adding new content

### DISCUSSION THREAD

**Discussion by tridoxx** [GetDKAN/dkan#3884] [Q&A]
Hello, I need your help please, to finish my school work, I can't find the other types of data creation, such as data history and visualization, is it deactivated or maybe it was deleted on dkanv2? I appreciate your help

link of dkan 1 where write about it 
https://dkan.readthedocs.io/en/latest/admin/data_and_content/adding_new_content.html

**Comment by janette** [comment]
@tridoxx do you see a "DKAN" menu at the top? If yes select "datasets" (/admin/dkan/datasets) and click the "Add dataset" button (/node/add/data)

[DKAN v2 docs
](https://demo.getdkan.org/modules/contrib/dkan/docs/guide_dataset.html)
![menu](https://user-images.githubusercontent.com/314172/206304710-842eccd7-e2b9-457a-89f3-70faf1d64022.png)

**Comment by tridoxx** [comment]
hello, yes, thanks, it wasn't about adding a dataset, it was to add a text, for example, or an article, and that this is within the filters of the dkan dataset search engine, I did a little trick, I added a text field, in the content called "data" , and you can now add a "dataset" without information but with text, the idea is to be able to apply the filters, such as theme, keyword, or publisher name, with that I did it and thank you very much for the help I think I'm done the project, I am going to answer my previous doubts so that anyone who has them can solve them
